Tasks/GooglePlayReleaseV4/modules/metadataHelper.ts

```diff
@@ -1,29 +1,41 @@
 import * as fs from 'fs';
 import * as path from 'path';
+import * as jsdom from 'jsdom';
 import * as tl from 'azure-pipelines-task-lib/task';
 
 import { androidpublisher_v3 as pub3 } from 'googleapis';
 
 /**
  * Uploads change log files if specified for all the version codes in the update
- * @param changelogFile
- * @param versionCodes
- * @returns nothing
+ * @param {string} languageCode
+ * @param {string} changelogFile
+ * @param {boolean} releaseNotesContainLanguageTags
+ * @returns {pub3.Schema$LocalizedText[]} `[{ language: 'en-US|fr-FR|it-IT|...', text: 'Localized_Release_Notes' }, ...]`
  */
-export async function getCommonReleaseNotes(languageCode: string, changelogFile: string): Promise<pub3.Schema$LocalizedText | null> {
+export async function getCommonReleaseNotes(languageCode: string, changelogFile: string, releaseNotesContainLanguageTags: boolean): Promise<pub3.Schema$LocalizedText[]> {
     const stats: tl.FsStats = tl.stats(changelogFile);
+    let releaseNotes: pub3.Schema$LocalizedText[] = [];
 
-    let releaseNotes: pub3.Schema$LocalizedText = null;
     if (stats && stats.isFile()) {
         console.log(tl.loc('AppendChangelog', changelogFile));
-        releaseNotes = {
-            language: languageCode,
-            text: getChangelog(changelogFile)
-        };
-
+        const changelog: string = getChangelog(changelogFile);
+
+        if (changelog) {
+            releaseNotesContainLanguageTags ? new jsdom.JSDOM(changelog).window.document.body.childNodes.forEach(node => {
+                const language = node['tagName'];
+                const text = node.textContent.trim();
+                if (language && text) {
+                    releaseNotes.push({ language, text });
+                }
+            }) : releaseNotes.push({
+                language: languageCode,
+                text: changelog
+            });
```

Comment:
I think it's better to avoid the ternary operator in cases when there is no value assignment.

Can we use just `if(releaseNotesContainLanguageTags) / else` here?

Reply:
Done, thanks!